### PR TITLE
[GSoC] Function for plotting bending moment added

### PR DESCRIPTION
**Pull request by Psycho-Pirate**
#### References to other Issues or PRs

See #19202

#### Brief description of what is fixed or changed
A function for plotting Bending Moment in 3D Beam is added.

#### Release Notes

* physics.continuum_mechanics
    * Added ability to plot bending moment in Beam3D class

### CODE REVIEW

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -2344,3 +2344,102 @@ def plot_shear_force(self, dir="all", subs=None):
...
+            [0]: cartesian line: -15*x**2/2 for x over (0.0, 20.0)
+            Plot[2]:Plot object containing:
+            [0]: cartesian line: 2*x**3 for x over (0.0, 20.0)
+
+        """
+        Px = self._plot_bending_moment('x')
+        Py = self._plot_bending_moment('y')
+        Pz = self._plot_bending_moment('z')
+        # For bending moment along x direction
+        if dir == "x":
```
Comment (ishanaj) [inline sympy/physics/continuum_mechanics/beam.py:2435]: Possibly, you can convert the case of dir `dir = dir.lower()` just before you start the if statements, to handle a case where the user specifies `dir` in upper case (X/Y/Z).

File: sympy/physics/continuum_mechanics/beam.py [hunk 2]
```diff
@@ -2344,3 +2344,102 @@ def plot_shear_force(self, dir="all", subs=None):
...
+            [0]: cartesian line: 0 for x over (0.0, 20.0)
+            Plot[1]:Plot object containing:
+            [0]: cartesian line: -15*x**2/2 for x over (0.0, 20.0)
+            Plot[2]:Plot object containing:
+            [0]: cartesian line: 2*x**3 for x over (0.0, 20.0)
+
+        """
+        Px = self._plot_bending_moment('x')
+        Py = self._plot_bending_moment('y')
+        Pz = self._plot_bending_moment('z')
```
Comment (ishanaj) [inline sympy/physics/continuum_mechanics/beam.py:2433]: If the `dir` specified is 'x' then why bother calculating `Py` and `Pz`. I think the helper functions can be shifted directly inside the if statements. Will reduce the computation time of the function.

**Comment by ishanaj** [comment]
Also, can tests be added for these?

**Comment by Psycho-Pirate** [comment]
I have made the requested changes in all three PRs.
Here's an example I used in the documentation.
```
>>> from sympy.physics.continuum_mechanics.beam import Beam3D
>>> from sympy import symbols
>>> l, E, G, I, A, x = symbols('l, E, G, I, A, x')
>>> b = Beam3D(20, E, G, I, A, x)
>>> b.apply_load(15, start=0, order=0, dir="z")
>>> b.apply_load(12*x, start=0, order=0, dir="y")
>>> b.bc_deflection = [(0, [0, 0, 0]), (20, [0, 0, 0])]
>>> R1, R2, R3, R4 = symbols('R1, R2, R3, R4')
>>> b.apply_load(R1, start=0, order=-1, dir="z")
>>> b.apply_load(R2, start=20, order=-1, dir="z")
>>> b.apply_load(R3, start=0, order=-1, dir="y")
>>> b.apply_load(R4, start=20, order=-1, dir="y")
>>> b.solve_for_reaction_loads(R1, R2, R3, R4)
>>> b.plot_bending_moment()
```
3d_bending

**Review by ishanaj — APPROVED** [review]

**Comment by ishanaj** [comment]
The changes look good to me. Merging!

**Pull request merged**